bin/setup

```diff
@@ -26,13 +26,18 @@ FileUtils.chdir APP_ROOT do
   end
 
   puts "\n== Preparing database =="
-  system! "bin/rails db:drop db:create db:migrate"
+  output = `bin/rails db:drop db:create db:migrate`
+  puts output.lines.grep(/database|migrated/)
```

Comment:
This reduces the output of the `db:migrate` part to look like this:

```
== Preparing database ==
Dropped database 'example_app_development'
Dropped database 'example_app_test'
Created database 'example_app_development'
Created database 'example_app_test'
== 20200214200848 CreatePosts: migrated (0.0048s) =============================
== 20200223142854 CreateCategories: migrated (0.0029s) ========================
== 20200223143509 CreatePostCategories: migrated (0.0063s) ====================
== 20200224003435 AddDescriptionToPost: migrated (0.0012s) ====================
== 20200311125252 CreateMovies: migrated (0.0031s) ============================
== 20200313180126 ChangeMovieRevenueLimit: migrated (0.0034s) =================
== 20200327150102 CreateGenres: migrated (0.0028s) ============================
== 20200327150540 CreateMovieGenres: migrated (0.0048s) =======================
== 20200508143117 CreateUsers: migrated (0.0032s) =============================
== 20200508143205 AddAuthorToPost: migrated (0.0045s) =========================
```

instead of showing the details of each step.